src/main/resources/db/migration/V134__land_i_medlemskap.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE soknadsskjema ADD COLUMN medlemskap_oppholdsland VARCHAR DEFAULT NULL;
+ALTER TABLE soknad_utenlandsopphold ADD COLUMN land VARCHAR;
```

Comment:
Noen grunn til vi kun har `DEFAULT NULL` på den første? Mulig vi ikke trenger det på noen av dem hvis testene gikk fint 🤔

Reply:
Litt usikker på om det blir riktig tankegang, men det jeg tenkte var at dersom noe skal legges til i utenlandsopphold så er det fordi en hel rad skal fylles. Det skal ikke være mulig å legge til en ny rad hvor den er null. Mens den første skal være null i de fleste tilfelle fordi man oppholder seg i norge

Reply:
En mulighet her er å lagre ned norge som oppholdsland i stedet for null, men det blir ikke så lett å etterfylle eksisterende rader fordi man vet ikke landet for de som ikke er i norge